client/installation-helper.cc

```diff
@@ -195,6 +195,39 @@ step4()
     cout << "done" << endl;
 }
 
+void
+step5(const string& description)
+{
+    // fate #317973
+    // step runs in chroot
+
+    // preconditions (maybe incomplete):
+    // snapper rpms installed in chroot
+
+    unsigned int num;
+
+    SCD scd;
+    scd.read_only = false;
+    scd.description = description;
+
+    Snapper snapper("root", "/");
```

Comment:
The root_prefix must be flexible so that the tool can also be used for the update case where the "pre" snapshot is done with the binary from the installation medium.